assets/wizards/engagement/index.js

```diff
@@ -147,6 +152,19 @@ class EngagementWizard extends Component {
 								) }
 							/>
 						) }
+						{ newspack_engagement_wizard.has_reader_activation && (
+							<Route
+								path="/reader-activation/complete"
+								render={ () => (
+									<ReaderActivationComplete
+										subHeaderText={ __(
+											'Preview and customize the reader activation prompts',
+											'newspack'
+										) }
```

Comment:
If you add a `{ ...props }` here it would bring the tabbed navigation to this route from [this earlier line](https://github.com/Automattic/newspack-plugin/blob/master/assets/wizards/engagement/index.js#L107-L110).

Reply:
Updated in 82c8612